index.html

```diff
@@ -3,27 +3,101 @@
 <head>
     <meta charset="UTF-8">
     <title>Player cards by Krzysztof</title>
+    <link rel="stylesheet" href="style.css" type="text/css" />
+    <link href="https://fonts.googleapis.com/css2?family=Work+Sans:wght@500;700&display=swap" rel="stylesheet" />
 </head>
 <body>
     <header>
-        <h1>This is my first madafakin' great project</h1>
+        <h1>This is my first madafakin' great project <br />
+            FIFA Players' cards
+        </h1>
     </header>
-    <nav class="navigation">
-        <button class="button button__team-comparison-page" type="button">
-            <a href="team_comparison_page.html">Go to creator</a>
-        </button>
+    <nav class="navigation main_navigation">
+        <div class="button button__team-comparison-page" type="button">
+            <a href="team_comparison_page/team_comparison_page.html">Go to team comparison</a>
+        </div>
     </nav>
     <section class="best-player-cards">
-        <h1>Best players' cards</h1>
-        <div class="best-player-cards__player-card">Place for card 1</div>
-        <div class="best-player-cards__player-card">Place for card 2</div>
-        <div class="best-player-cards__player-card">Place for card 3</div>
-        <div class="best-player-cards__player-card">Place for card 4</div>
+        <div class="best_players_cards__header">Best players' cards</div>
+        <div class="best_players_cards__cards">
+            <div class="player-card player-card__best-player-card">
+                <div class="player_card__photo">
+                    <img src="assets/player-photos/KZ.png" style="float: top">
```

Comment:
move style to code, do not use inline style, it will increase complexity of maintain code in future

Reply:
similar mistakes in bellow code